### PR TITLE
[vhost.sh] Fixed conditional commented lines

### PR DESCRIPTION
**Pull request by vidarl**
Fixes for vhost.sh

This is based on AR's travis-docker branch and I use the same concepts.

Changes:
- Removes "if[var...]" in output ( lines remains ordinary comments without our "if" statement
- Fixed "#if[VAR!=some-other-value] " code so it now works with multiple occurences

### CODE REVIEW

File: bin/vhost.sh [hunk 1]
```diff
@@ -236,24 +236,42 @@ while [  "${template_vars[$COUNTER]}" != "" ]; do
...
+    # Remove "%" from VAR for further use
+    current_var=${current_var//%/}
+
     # If variable has a value then do further replacment logic
     if [ "$current_value" != "" ] ; then
-        # Remove "%" from VAR for further use
-        current_var=${current_var//%/}
 
-        # Remove "#if[VAR] " comments to conditionally uncomment lines
+        # Change "#if[VAR] " comments conditionally to no-comment lines
```
Comment (andrerom) [inline bin/vhost.sh]: never heard the term "no-comment" about uncommenting a line :)

**Comment by andrerom** [comment]
+1 besides maybe some improvements to the inline comments.

File: bin/vhost.sh (hunk 1 above)
Comment (vidarl) [inline bin/vhost.sh]: I didn't feel "uncomment lines" sounded that good either, but "no-comment lines" was maybe not an improvement.

You want it back to "uncomment lines", or is "uncommented lines" better as in :
`+        # Change "#if[VAR] " comments conditionally to uncommented lines`

Comment (andrerom) [inline bin/vhost.sh]: Your new proposal looks good.

**Comment by vidarl** [comment]
> +1 besides maybe some improvements to the inline comments.

Fixed

**Pull request merged**